### PR TITLE
Tjurina module 

### PR DESCRIPTION
**Pull request by Syz-MS**
Adds Tjurina module and number for CompleteIntersectionGerm.

@afkafkafk13 please review.

### CODE REVIEW

File: experimental/Schemes/src/Tjurina.jl [hunk 1]
```diff
@@ -738,3 +738,88 @@ function is_contact_equivalent(X::HypersurfaceGerm, Y::HypersurfaceGerm)
...
+2 -> e[2]
+by Submodule with 7 generators
+1 -> 2*x*e[1] + y*e[2]
+2 -> 2*y*e[1] + x*e[2]
+3 -> -2*z*e[1]
+4 -> (x^2 + y^2 - z^2)*e[1]
+5 -> (x^2 + y^2 - z^2)*e[2]
+6 -> x*y*e[1]
+7 -> x*y*e[2]
+```
```
Comment (afkafkafk13) [inline experimental/Schemes/src/Tjurina.jl]: please add a call of monomial_basis to this example for users' convenience.

File: experimental/Schemes/src/Tjurina.jl [hunk 2]
```diff
@@ -738,3 +738,88 @@ function is_contact_equivalent(X::HypersurfaceGerm, Y::HypersurfaceGerm)
...
+
+#####                           Tjurina module                             #####
+
+################################################################################
+
+
+@doc raw"""
+    tjurina_module(X::CompleteIntersectionGerm) 
+
+Return the Tjurina module of the complete intersection germ `(X,p)` at the point `p`.
```
Comment (afkafkafk13) [inline experimental/Schemes/src/Tjurina.jl:755]: Maybe add a note that this is a special case of the T^1. I assume that T^1 and T^2 in the general case are also somewhere further behind in the pipeline?

**Review by afkafkafk13 — COMMENTED** [review]
Looks good. Thank you. I left you some remarks on the docstrings.

File: experimental/Schemes/src/Tjurina.jl (hunk 2 above)
Comment (Syz-MS) [reply experimental/Schemes/src/Tjurina.jl:755]: Will be added to doc when T^1 and T^2 are implemented.

**Review by afkafkafk13 — APPROVED** [review]
Good to go, as soon as the tests have passed.

**Comment by afkafkafk13** [comment]
@Syz-MS : Failure of the test 1.6, long. ubuntu-latest is due to a problem on the underlying system, not due to you. I will restart the job after all the others have passed.

**Comment by joschmitt** [comment]
> @Syz-MS : Failure of the test 1.6, long. ubuntu-latest is due to a problem on the underlying system, not due to you. I will restart the job after all the others have passed.

I restarted the job now.

**Pull request merged**